js/ui/validator.js

```diff
@@ -72,118 +73,116 @@ var Validator = DOMComponent.inherit({
             * @type_function_param1_field1 name:string
             * @type_function_param1_field2 isValid:boolean
             * @type_function_param1_field3 value:Object
-            * @type_function_param1_field4 validationRules:Array<RequiredRule,NumericRule,RangeRule,StringLengthRule,CustomRule,CompareRule,PatternRule,EmailRule>
-            * @type_function_param1_field5 brokenRule:RequiredRule|NumericRule|RangeRule|StringLengthRule|CustomRule|CompareRule|PatternRule|EmailRule
+            * @type_function_param1_field4 validationRules:Array<RequiredRule,NumericRule,RangeRule,StringLengthRule,CustomRule,CompareRule,PatternRule,EmailRule,AsyncRule>
+            * @type_function_param1_field5 brokenRule:RequiredRule|NumericRule|RangeRule|StringLengthRule|CustomRule|CompareRule|PatternRule|EmailRule|AsyncRule
+            * @type_function_param1_field6 brokenRules:Array<RequiredRule,NumericRule,RangeRule,StringLengthRule,CustomRule,CompareRule,PatternRule,EmailRule,AsyncRule>
+            * @type_function_param1_field6 status:Enums.ValidationStatus
             * @action
             */
 
             /**
             * @name dxValidatorOptions.rtlEnabled
             * @hidden
+            * @inheritdoc
```

Comment:
Is it neccessary?

Reply:
No. It seems to be merged accidentally from previous commits.